<launch>
  <!-- Load Id Map configs -->
  <include file="$(find humanoid_loadmap)/launch/sakura_loadmap.launch"/>
  <!-- Interface -->

  <arg name="device_name"      default="/dev/ttyUSB0"/>
  <arg name="baud_rate"        default="1000000"/>
  <arg name="protocol_version" default="2.0"/>
  <arg name="toRobot"          default="false"/>
  <arg name="toGazebo"         default="true"/>
  <arg name="dt"               default="0.07"/>

  <param name="humanoid_interface/device_name"      value="$(arg device_name)"/>
  <param name="humanoid_interface/baud_rate"        value="$(arg baud_rate)"/>
  <param name="humanoid_interface/protocol_version" value="$(arg protocol_version)"/>
  <param name="humanoid_interface/toRobot"          value="$(arg toRobot)"/>
  <param name="humanoid_interface/toGazebo"         value="$(arg toGazebo)"/>
  <param name="humanoid_interface/dt"               value="$(arg dt)"/>


  <node name="humanoid_interface" pkg="humanoid_interface" type="humanoid_interface_node" output="screen" >   
  </node>
</launch>
